### PR TITLE
Added requester instructions to README.md

### PR DESCRIPTION
**Pull request by shreyasshivakumara**
Requester Instructions on how to download and decrypt the data files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,3 +130,49 @@ following command:
...
+
+The first step is to generate a keypair using the `crypt4gh` encryption tool. This can be done using the following command. There are two keys generated - `user.sec` is the secret key and `user.pub` is the public key. You must reply to the FEGA-SE helpdesk by sending them the generated public key - `user.pub`. 
+
+```bash
+crypt4gh-keygen --sk user.sec --pk user.pub
+```
+
+### Decrypting the data files
+
+You recieve an email from the FEGA-SE helpdesk that contains a text file. This text file contains links to download the encrypted data files. Download the download_data script from [here](https://raw.githubusercontent.com/NBISweden/LocalEGA-SE-Deployment/master/dev-tools/publicize_file/download_data.sh?token=GHSAT0AAAAAABQNNF76ZH6NKQZFOOGELTXQYPOT6FA) and use the following command.
```
Comment (dbampalikis) [inline README.md]: I am not sure, but I think this token is private information and shouldn't be here. If we need the script for the users, we can move it to this repo

File: README.md [hunk 2]
```diff
@@ -130,3 +130,49 @@ following command:
...
+
+```bash
+./download_data.sh path/filename
+```
+This downloads all the encrypted files from the text file and allows it to maintains the structure of the dataset. Next, you must transfer the downloaded encrypt files to a secure environment. Use the following command to decypt files inside the secure environment using the secret key - `user.sec` that was generated in the previous step.
```
Comment (dbampalikis) [inline README.md]: Maybe mention that the user needs to transfer the private key as well to the secure environment

File: README.md (hunk 1 above)
Comment (kostas-kou) [inline README.md]: I think that it was mentioned that the helpdesk will send the link and not the text file itself

File: README.md (hunk 2 above)
Comment (kostas-kou) [inline README.md]:
```suggestion
This downloads all the encrypted files from the text file and allows it to maintains the structure of the dataset. Next, you must transfer the downloaded encrypt files to a secure environment. Use the following command to decrypt files inside the secure environment using the secret key - `user.sec` that was generated in the previous step.
```

File: README.md (hunk 1 above)
Comment (shreyasshivakumara) [reply README.md]: Just resolved it. Thanks!

File: README.md [hunk 3]
```diff
@@ -130,3 +130,49 @@ following command:
...
+
+The first step is to generate a keypair using the `crypt4gh` encryption tool. This can be done using the following command. There are two keys generated - `user.sec` is the secret key and `user.pub` is the public key. You must reply to the FEGA-SE helpdesk by sending them the generated public key - `user.pub`. 
+
+```bash
+crypt4gh-keygen --sk user.sec --pk user.pub
+```
+
+### Decrypting the data files
+
+You recieve an email from the FEGA-SE helpdesk that contains a text file. This text file contains links to download the encrypted data files. Download the download_data script from [here](https://github.com/NBISweden/LocalEGA-SE-Deployment/blob/refactor/one-checksum-file/dev-tools/publicize_file/download_data.sh) and use the following command.
```
Comment (dbampalikis) [inline README.md]: We need to move the `download_data.sh` to the public repo, so that the users have access to it

**Review by kostas-kou — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (shreyasshivakumara) [reply README.md]: Yes, that's the plan. Currently, for the demonstrator, we plan to use this.

**Review by jmenglund — APPROVED** [review]

**Review by kostas-kou — APPROVED** [review]
You should remove the link with the token from history as well

**Pull request merged**